pubspec.yaml

```diff
@@ -3,15 +3,16 @@ version: 1.1.0
 description: Command line interface for the Dart Analyzer.
 author: Dart Team <misc@dartlang.org>
 homepage: https://github.com/dart-lang/analyzer_cli
+environment:
+  sdk: '>=1.8.0 <2.0.0'
```

Comment:
You'd think.  It's a bootstrapping thing.  They could have done this differently but the idea is that the one place you can't use these clever constraints is if you're testing if you *can* use these smart constraints (e.g., `sdk >= 1.8.0`).